src/pl.x86.json

```diff
@@ -523,6 +523,16 @@
 			"author": "Jon Galletero",
 			"homepage": "https://sourceforge.net/projects/nppmusicplayer"
 		},
+		{
+			"folder-name": "Remove_dup_lines",
+			"display-name": "Remove duplicate lines",
```

Comment:
- List is sorted by display-name, so this is not the right place, but between Q and S.
- Zip has still not in the expected structure that the dll is directly at the toplevel inside of the zip